src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -2,6 +2,7 @@ const MetaTraderConfig   = require('./metatrader.config');
 const MetaTraderUI       = require('./metatrader.ui');
 const Client             = require('../../../base/client');
 const BinarySocket       = require('../../../base/socket');
+const DerivBanner        = require('../../../common/deriv_banner');
```

Comment:
I would import the ClientBase here aswell then, to make use of the getAccountType function.